MapboxVisionTests/RecordCoordinatorTests.swift

```diff
@@ -72,6 +70,70 @@ final class RecordCoordinatorTests: XCTestCase {
         XCTAssert(directoryExists(at: recordingPath), "Recording should be saved at \(recordingPath) after recording is stopped")
     }
 
+    func testVideoClipping() {
+        // Given
+        // a list of clip requests
+        // When
+        // we try to make clips with them
+        // Then
+        // we should get the same number of video files on filesystem
```

Comment:
Probably a bit confusing comment - could you please take a look at comment for `performVideoClipping` func?